packages/visualizations-stories/src/grid/2-table-grid.stories.tsx

```diff
@@ -82,7 +103,8 @@ storiesOf("@operational/grid/2. Table", module).add("basic", () => {
           height={height}
           data={frame}
           style={{
-            cell: { padding: "10px" },
+            cell: (rowIndex, columnIndex) => ({ padding: "10px", background: columnIndex > 0 ? colors[frame.row(rowIndex)[1] as "Germany" | "UK" | "USA" | "Canada"] : "#fff" }),
```

Comment:
```suggestion
            cell: (rowIndex) => ({ padding: "10px", background: colors[frame.row(rowIndex)[countryCursor.index] as "Germany" | "UK" | "USA" | "Canada"] }),
```

- as we discussed: it would make more sense to highlight whole row, because row corresponds to point in N-dimensional space
- use `countryCursor` instead of hardcoded number to not encourage this style of code

Reply:
I think we should still allow the user to colour by either row or column index - there are clear use cases for this, especially once we allow the user to apply a colour scale or gradient to a measure:

![image](https://user-images.githubusercontent.com/14272822/62031584-04a59400-b1e8-11e9-9f75-76e599a7ec9c.png)

The user might even just want to highlight a single cell.

Reply:
> there are clear use cases for this, especially once we allow the user to apply a colour scale or gradient to a measure:

my point is that there is no use-case for that in table, because each row is one point in N-Dimension. By using more than one colour we assign more than one colour to the point, which is only possible in table, but not possible in pivot neither in any chart

Reply:
Ah, gotcha.

Reply:
So are we leaving it as is?

Reply:
Yes let's leave it